_source/_posts/2024-02-29-3pc.md

```diff
@@ -0,0 +1,65 @@
+
+third-party cookies allow user behavior to be tracked across different sites which they visit. These cookies are now creatively misused by advertisers and trackers to exploit personal information. For the legitimate tasks which used to require third-party cookies, like social integration and caching, there are now better and more privacy-respecting technologies available. 
+
+Today, the drawbacks to users' security and privacy from third-party cookie implementations outweigh their benefits so much that all major browsers are phasing them out. Firefox, Edge, and Safari have been blocking third-party cookies for years, and Chrome is now phasing them out in 2024. 
+
+If your code uses Okta features that rely on third-party cookies, this means that you'll need to make some changes to keep the identity experience working as intended. 
+
+
+## Does your application use third-party cookies? 
+
+Most code using Okta doesn't use third-party cookies, and won't be affected by the deprecation. Here are the design patterns in which Okta uses third-party cookies. If your application is in one of these categories, please test its behavior with third-party cookie deprecation. 
+
+Okta uses cookies to let applications introspect and extend user sessions. Cookies aren't required for basic login functionality. 
+
+### Web applications that rely on the Okta session for user context.  
+
+If an application hosted on your domain (`mycompanyapp.com`) redirects to your Okta subdomain (`mycompany.okta.com`) for login and then returns users to your own domain, third-party cookie restrictions will limit how your app can introspect or extend the Okta session. 
+
+
+### Customer-hosted Okta Sign-In Widget and customer-built login applications
+
+If you're hosting your own sign-in experience on a separate top level domain from your main app, you may be using third-party cookies. You might be hosting your own sign-in experience by cloning the Okta Sign-In Widget from GitHub or installing it from NPM to embed in your application, or you might have built your own custom sign-in using Okta's APIs. 
+
+If your sign-in experience is hosted on the same top-level domain as your application, third-party cookie deprecation won't affect its behavior. 
+
+If the sign-in experience and app are on different top-level domains, third-party cookie deprecation will break its ability to introspect and extend sessions, because these features use cookies. Authentication will still be possible, and tokens will still be returned, becasue these features do not rely on cookies. 
+
+### "Remember me" Features
+
+"Remember Last Used Factor" (RLUF), "Keep me signed in" (OIE) and "Remember me" (Classic) will not function when the login application is on a third-party domain.  
```

Comment:
These are examples of an auth flow functioning, but with a degraded experience

(see "Most code using Okta doesn't use third-party cookies, and won't be affected by the deprecation." comment)

Reply:
I rewrote the "most code..." bit to frame it as "login doesn't break but certain experience stuff may", so I think that probably resolves this. Thanks for calling that out!